src/features/processPicker.ts

```diff
@@ -452,4 +452,4 @@ function execChildProcessAndOutputErrorToChannel(process: string, workingDirecto
         });
     });
 
-}
\ No newline at end of file
+}
```

Comment:
This is just a whitespace change. 

---
In reply to: [89404046](https://github.com/OmniSharp/omnisharp-vscode/pull/983#discussion_r89404046) [](ancestors = 89404046)